lib/seattleflu/id3c/cli/command/etl/redcap.py

```diff
@@ -688,3 +688,16 @@ def build_questionnaire_items(question: str) -> Optional[dict]:
         return create_resource_entry(questionnaire_reseponse_resource, full_url)
 
     return None
+
+
+def extract_date_from_survey_timestamp(record: REDCapRecord, survey_name: str) -> Optional[str]:
+    """
+    Extracts as a string the date component of the *survey_name* REDCap survey timestamp, the system
+    timestamp that is captured automatically and is not dependent on the client. This timestamp
+    is in local (Pacific) time.
+    """
```

Comment:
It might be worth to add a comment to highlight the null edge case with instrument timestamps that Redcap notes at the end of the api documentation entry,

(From https://redcap.iths.org/redcap_v10.5.2/API/help.php?content=exp_records )

exportSurveyFields:
true, false [default] - specifies whether or not to export the survey identifier field (e.g., 'redcap_survey_identifier') or survey timestamp fields (e.g., instrument+'_timestamp') when surveys are utilized in the project. If you do not pass in this flag, it will default to 'false'. If set to 'true', it will return the redcap_survey_identifier field and also the survey timestamp field for a particular survey when at least one field from that survey is being exported. **NOTE: If the survey identifier field or survey timestamp fields are imported via API data import, they will simply be ignored since they are not real fields in the project but rather are pseudo-fields.**

Reply:
I added in that note.